themes/godotengine/layouts/features.htm

```diff
@@ -185,8 +185,8 @@ <h2>Multi-platform editor</h2>
       <h2>Multi-platform deploy</h2>
       <em>Deploy games everywhere!</em>
       <ul>
-        <li><b>Mobile platforms</b>, iOS, Android, BlackBerry OS</li>
-        <li><b>Desktop platforms</b> Windows (also UWP), OS X, Linux, *BSD and Haiku</li>
+        <li><b>Mobile platforms:</b> iOS, Android, BlackBerry OS</li>
```

Comment:
Actually BlackBerry is not supported in 3.0, could you remove the mention to it?